.travis.yml

```diff
@@ -15,7 +15,7 @@ branches:
     - /^\d+\.\d+(\.\d+)?(-\S*)?$/
 
 env:
-  - BAZEL=0.13.0 TF=NIGHTLY
+  - BAZEL=0.16.1 TF=NIGHTLY
```

Comment:
Can you add a comment to keep this in sync with versions.check in the WORKSPACE file, and a symmetrical comment there about .travis.yml?

Reply:
Sure.